packages/app/src/systems/Bridge/types.tsx

```diff
@@ -14,3 +14,10 @@ export type BridgeTx = {
   toNetwork: SupportedChain;
   isDone: boolean;
 };
+
+export type BridgeAsset = {
+  address?: string;
+  decimals?: number;
+  symbol?: string;
```

Comment:
should these 3 be required?  at the very least address should not be able to be set to undefined

Reply:
nvm i see that we leave address undefined for eth

Reply:
address should accept undefined.in eth side, ETH has no address because it's the native asset. only erc20 will have addresses.

in fuel side, native asset will also have address.

but as this is a generic asset type for all chains, we have all fields accepting `undefined` to improve flexibility across chains